### PR TITLE
Added Strike Bowling Bar

### PR DESCRIPTION
**Pull request by GenericError**
Created brands/leisure/bowling_alley.json and added an entry for Strike Bowling Bar

### CODE REVIEW

**Comment by GenericError** [comment]
Related issue #3119

File: brands/leisure/bowling_alley.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "leisure/bowling_alley|Strike Bowling Bar": {
+    "countryCodes": ["au"],
+    "matchTags": ["leisure/bowling_alley"],
```
Comment (bhousel) [inline brands/leisure/bowling_alley.json]: This `matchTags` property can be removed,  it already matches that one.

Comment (GenericError) [reply brands/leisure/bowling_alley.json]: Thanks for that; looking at the other files it looks unnecessary

**Comment by bhousel** [comment]
Thanks! looks good 👍

**Pull request merged**